corehq/sql_db/routers.py

```diff
@@ -66,8 +66,8 @@ def allow_migrate(db, app_label):
     :return: Must return a boolean value, not None.
     """
     if app_label == ICDS_REPORTS_APP:
-        db_alias = get_icds_ucr_db_alias()
-        return bool(db_alias and db_alias == db)
+        db_aliases = [get_icds_ucr_db_alias(), get_icds_ucr_citus_db_alias()]
+        return bool(db in db_aliases)
```

Comment:
nit: does this need `bool`. i think `x in y` returns a boolean value

Reply:
```suggestion
        return db in db_aliases
```